task4/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace task4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double x = Funk(1,3,2) + Funk(2,5,7) + Funk(3,3,6);
+            Console.WriteLine(Math.Round(x,3));
```

Comment:
Лучше не округлять, а форматировать
```suggestion
            Console.WriteLine($x = {x:F3}");
```